TinyMCE.Blazor/wwwroot/tinymce-blazor.js

```diff
@@ -1,4 +1,4 @@
-console.log('loading js tinymce-blazor');
+﻿console.log('loading js tinymce-blazor');
```

Comment:
Another `console.log` that may be for debugging.

Reply:
Same as above